packages/web-app/src/modules/versions/VersionStore.ts

```diff
@@ -45,13 +45,13 @@ export class VersionStore {
       removeItem(VERSION_RELOAD_DATA)
     }
 
-    if (this.store.native.isNative) {
-      this.checkDesktopVersion()
-    }
-
     autorun(() => {
+      if (this.latestDesktopVersion === undefined) {
```

Comment:
Should we only make the API request if we know we're in the desktop app?

```suggestion
      if (this.latestDesktopVersion === undefined && this.store.native.isNative) {
```

If yes, may want to refactor this since the next block also appears to require being in the desktop app. Even better-- could we skip creating the `autorun` if we're not in the desktop app? One less function being called!